lis/core/LIS_histDataMod.F90

```diff
@@ -4494,6 +4535,215 @@ subroutine LIS_histDataInit(n, ntiles)
             n, 1, ntiles,(/"g/m2"/), 1, (/"-"/),1,1,1,&
             model_patch=.true.)
     endif
+
+    ! AquaCrop
```

Comment:
Just my own curiosity: is this the same collection of output as in the default AquaCrop simulations, or already customized?

Reply:
Good question: those are variables that I selected. Sometimes, the default AquaCrop output (written out in the .OUT files) do not directly correspond to a global variable (i.e. some calculations are still performed in the writing of the output). I could add more but the user will likely select its own.

Reply:
What do you mean by 'some calculations' are still performed? Are they important to be made also before writing output from LIS-AC?

Reply:
For example, some values (i.e. SM) are converted to percentages, other output variables are derived from other, e.g. in run.f90
            HI = 100._sp * (GetSumWaBal_YieldPart())/(GetSumWaBal_Biomass())
So when a user wants to output another variable, he really has to look into the WriteDailyResults procedure to make sure he outputs the same thing. 

I could try to align the LIS output with the standalone but that would take me another (1/2) day.

Reply:
Discussed with Michel: keep it in line with LIS variables (e.g. soil moisture not as percentages) and keep it as is.